lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -200,7 +200,7 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
 
   defp edit_sessions(assigns) do
     ~H"""
-    <div class="mx-4 mr-2 text-gray-600 flex flex-row gap-1">
+    <div class="mx-4 mr-2 text-gray-600 flex flex-row gap-1" phx-update="ignore">
       <.menu id="edit-sessions">
```

Comment:
We need an `id` too. Perhaps we can move it up here?

```suggestion
    <div class="mx-4 mr-2 text-gray-600 flex flex-row gap-1" id="edit-sessions" phx-update="ignore">
      <.menu>
```

Reply:
The menu requires an `id`, so we need both :)

Reply:
The accessibility too!